storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -48,5 +48,10 @@ To remove a third-party CSI driver, see xref:../../storage/container_storage_int
 
 include::modules/persistent-storage-csi-vsphere-install-issues.adoc[leveloffset=+1]
 
+include::modules/persistent-storage-csi-vsphere-top-aware.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
```

Comment:
There are two "Additional resources" sections here. Should we combine them?

Reply:
It's kind of weird, but the one add resources is for the module, and the other one is for the whole topic. Geez, I hate that we can't just xref in modules. It creates this kind of weirdness.